assembly/assembly-ide-war/pom.xml

```diff
@@ -22,7 +22,7 @@
         <groupId>com.codenvy.onpremises</groupId>
         <version>5.3.0-SNAPSHOT</version>
     </parent>
-    <artifactId>onpremises-ide-packaging-war-ide</artifactId>
+    <artifactId>assembly-ide-war</artifactId>
```

Comment:
Take a note, that if you have _che_ and _codenvy_ configured in intellij or smth different IDE, it will difficult to find corresponding module to launch super dev mode.

![2017-02-03 16 18 52](https://cloud.githubusercontent.com/assets/1968177/22594691/c665a354-ea2c-11e6-9ddc-bdbe6739e6a7.png)